x/posts/internal/types/post.go

```diff
@@ -132,7 +144,7 @@ func (p Post) String() string {
 // Validate implements validator
 func (p Post) Validate() error {
 	if !p.PostID.Valid() {
-		return fmt.Errorf("invalid post id: %s", p.PostID)
+		return fmt.Errorf("invalid post newID: %s", p.PostID)
```

Comment:
I think this is a refactoring error

```suggestion
		return fmt.Errorf("invalid post id: %s", p.PostID)
```